test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
@shabanovd No tests should have external web dependencies as it has caused us pain in the past. That above `org.exist.xquery.XQueryTest` should be fixed in a separate PR.